modules/codegen/src/test/scala/core/issues/Issue255.scala

```diff
@@ -23,6 +23,11 @@ class Issue255 extends FunSuite with Matchers with SwaggerSpecRunner {
                            |      somePassword:
                            |        type: string
                            |        format: password
+                           |      someFile:
```

Comment:
These are described in the comments of #255, could you edit the summary in the issue to describe the full scope of the issues that are tracked in it for reference?

If this regression test starts failing due to `type: file` starting to break and they see "Unknown type for the following structure (type: string, class: io.swagger.v3.oas.models.media.PasswordSchema)" I suspect they would be confused.

Reply:
will do